pkg/util/validation/trustbundle.go

```diff
@@ -27,53 +22,27 @@ func TrustBundleConfigMap(cfgMap *corev1.ConfigMap, caDataKey string) ([]*x509.C
 	if len(trustBundleData) == 0 {
 		return nil, nil, fmt.Errorf("data key %q is empty from ConfigMap %q", names.TRUSTED_CA_BUNDLE_CONFIGMAP_KEY, cfgMap.Name)
 	}
-	certBundle, _, err := CertificateData(trustBundleData)
+	certBundle, err := crypto.CertsFromPEM(trustBundleData)
```

Comment:
"This change addresses the problem by using crypto.CertsFromPEM from library-go, which returns early when it cannot decode the next PEM block without throwing an error."

So it returns early and has no error? What if the first PEM block is the invalid one?